pipelines/optimus/Optimus.wdl

```diff
@@ -49,9 +49,6 @@ workflow Optimus {
     
     # Emptydrops lower cutoff
     Int emptydrops_lower = 100
-    
```

Comment:
We don't want this override anymore?

Reply:
This was declared in duplicate. I noticed when I run mini-wdl on the file before checking in and fixed in the process. It's still there a bit further up. Highlights the need for a ticket I recently put in to run our now wdl 1.0 'lintable'pipelines through checks for syntax etc...